.env.example

```
@@ -1,13 +1,12 @@
 # JWT and API Settings
-APP_JWT_SECRET=
-APP_ENCRYPTION_KEY=
-APP_DATABASE_ENCRYPTION_KEY=
+APP_JWT_SECRET=""
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)